lib/pr_checker.js

```diff
@@ -203,7 +203,7 @@ class PRChecker {
       cli.error('No CI runs detected');
       this.CIStatus = false;
       return false;
-    } else if (!ciMap.get(FULL)) {
+    } else if (!ciMap.get(FULL) && !ciMap.get(LITE)) {
       status = false;
       cli.error('No full CI runs detected');
```

Comment:
Can you update this to `No full CI runs or lite CI runs detected`?

Reply:
Done.